explorer/test/Test/Pos/Explorer/Web/ServerSpec.hs

```diff
@@ -142,8 +137,13 @@ blocksPagesTotalUnitSpec =
                               $ getBlocksPagesTotal (Just 10)
 
                   -- We finally run it as @PropertyM@ and check if it holds.
-                  blocksTotal <- run blockExecution
-                  assert $ blocksTotal > 0
+                  pagesTotal <- run blockExecution
+
+                  -- And we assert that the generated blockchain total block pages count
+                  -- is equal to the expected explorer API result, which is number of
+                  -- generated blocks divided by 10.
+                  -- It shows that two equal algorithms should work the same.
+                  assert $ pagesTotal == (fromIntegral ((totalBlocksNumber - 1) `div` 10) + 1)
```

Comment:
Use `divRoundUp` from `Pos.Util.Util` instead of putting same algorithms here. With an unit test of `divRoundUp` we can make sure that `divRoundUp` will work here or anywhere.

Reply:
The comment is - `It shows that two equal algorithms should work the same.`, that does what it should - the function uses `divRoundUp`, the test uses a different algorithm that should work the same.

Reply:
Don't worry, I've read your comment :) However, I don't see any reason why we can't use `divRoundUp` here, if `divRoundUp` is (or will be) well tested.

Reply:
Because then we aren't testing anything. We are just showing that the result of the function1 is function1, which is a futile test.